# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: Cargo.toml
# Commit: da557d1806dd
# Full Hash: da557d1806dd9e1e519e00a4481b2e075f722ae8
# Author: Butkovits Atila <abutkovits@mozilla.com>
# Date: 2023-01-27 22:54 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing bustages complaining about AVIFDecodedData. CLOSED TREE
# ==============================================================================

diff -r d980bebc6d26 -r da557d1806dd Cargo.toml
--- a/Cargo.toml	Fri Jan 27 20:35:34 2023 +0000
+++ b/Cargo.toml	Fri Jan 27 22:54:08 2023 +0200
@@ -104,6 +104,9 @@
 # Patch getrandom 0.7 to 0.8
 rand = { path = "build/rust/rand" }
 
+# Patch env_logger 0.8 to 0.9
+env_logger = { path = "build/rust/env_logger" }
+
 # Patch parking_lot 0.12 down to 0.11, which is compatible for most crates that use it, to avoid
 # dependencies on windows-sys.
 parking_lot = { path = "build/rust/parking_lot" }